packages/vite-plugin-svelte/src/index.ts

```diff
@@ -85,7 +85,7 @@ export function svelte(inlineOptions?: Partial<Options>): Plugin[] {
 			},
 
 			async buildStart() {
-				if (!options.experimental?.prebundleSvelteLibraries) return;
+				if (!options.prebundleSvelteLibraries) return;
```

Comment:
When resolving the v-p-s options, should we auto-set `options.prebundleSvelteLibraries` if `options.experimental.prebundleSvelteLibraries` is set, and a warning? Not necessarily needed since it was an experimental option, but would be a nice-to-have for existing users who turn this on.

Reply:
yes, good idea. I've added that to the `config` hook